docs/changelog.md

```diff
@@ -112,7 +121,15 @@ Legend:
 
 > ➕ Added:
 >
-> 1. New and updated [resource types](./open-data/README.md#-resource-types) and icons.
+> 1. Added ServiceModel and Environment columns to the [services](./open-data/README.md#-services) data.
+> 2. New and updated [resource types](./open-data/README.md#-resource-types) and icons.
+
+🖥️ PowerShell
+{: .fs-5 .fw-500 .mt-4 mb-0 }
+
```

Comment:
I wasn't the author of this change. Consider validating whether this should be included.